packages/material-ui/src/styles/muiStyled.d.ts

```diff
@@ -198,10 +198,12 @@ export interface CreateStyled<Theme extends object = any> {
 }
 
 /**
- * Cutom styled functionality that support mui specific config.
+ * Cutom styled utility that has a default mui theme.
```

Comment:
*Nearly fixed. :)
```suggestion
 * Custom styled utility that has a default MUI theme.
```
@joshwooding For shame. :D